### PR TITLE
new tests in operators and new type for support bigDecimal in sums

### PR DESCRIPTION
**Pull request by gserranojc**

### CODE REVIEW

File: plugins/operator-accumulator/src/test/scala/com/stratio/sparkta/plugin/operator/accumulator/AccumulatorOperatorTestSpec.scala [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.stratio.sparkta.plugin.operator.accumulator
+
+import org.scalatest.{Matchers, WordSpec}
+
+class AccumulatorOperatorTestSpec extends WordSpec with Matchers {
```
Comment (danielcsant) [inline plugins/operator-accumulator/src/test/scala/com/stratio/sparkta/plugin/operator/accumulator/AccumulatorOperatorTestSpec.scala]: header anotation @RunWith(classOf[JUnitRunner]) missed

File: plugins/operator-avg/src/test/scala/com/stratio/sparkta/plugin/operator/avg/AvgOperatorSpec.scala [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.stratio.sparkta.plugin.operator.avg
+
+import org.scalatest.{Matchers, WordSpec}
+
+class AvgOperatorSpec extends WordSpec with Matchers {
```
Comment (danielcsant) [inline plugins/operator-avg/src/test/scala/com/stratio/sparkta/plugin/operator/avg/AvgOperatorSpec.scala]: header anotation @RunWith(classOf[JUnitRunner]) missed

File: plugins/operator-count/src/test/scala/com.stratio.sparkta.plugin.operator.count/CountOperatorSpec.scala [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.stratio.sparkta.plugin.operator.count
+
+import org.scalatest.{Matchers, WordSpec}
+
+class CountOperatorSpec extends WordSpec with Matchers {
```
Comment (danielcsant) [inline plugins/operator-count/src/test/scala/com.stratio.sparkta.plugin.operator.count/CountOperatorSpec.scala]: header anotation @RunWith(classOf[JUnitRunner]) missed

File: plugins/operator-fullText/src/test/scala/com/stratio/sparkta/plugin/operator/fullText/FullTextOperatorSpec.scala [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.stratio.sparkta.plugin.operator.fullText
+
+import org.scalatest.{Matchers, WordSpec}
+
+class FullTextOperatorSpec extends WordSpec with Matchers {
```
Comment (danielcsant) [inline plugins/operator-fullText/src/test/scala/com/stratio/sparkta/plugin/operator/fullText/FullTextOperatorSpec.scala]: header anotation @RunWith(classOf[JUnitRunner]) missed

File: plugins/operator-lastValue/src/test/scala/com/stratio/sparkta/plugin/operator/lastValue/LastValueOperatorSpec.scala [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.stratio.sparkta.plugin.operator.lastValue
+
+import org.scalatest.{Matchers, WordSpec}
+
+class LastValueOperatorSpec extends WordSpec with Matchers {
```
Comment (danielcsant) [inline plugins/operator-lastValue/src/test/scala/com/stratio/sparkta/plugin/operator/lastValue/LastValueOperatorSpec.scala]: header anotation @RunWith(classOf[JUnitRunner]) missed

File: plugins/operator-max/src/test/scala/com/stratio/sparkta/plugin/operator/max/MaxOperatorSpec.scala [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.stratio.sparkta.plugin.operator.max
+
+import org.scalatest.{Matchers, WordSpec}
+
+class MaxOperatorSpec extends WordSpec with Matchers {
```
Comment (danielcsant) [inline plugins/operator-max/src/test/scala/com/stratio/sparkta/plugin/operator/max/MaxOperatorSpec.scala]: header anotation @RunWith(classOf[JUnitRunner]) missed

File: plugins/operator-median/src/test/scala/com/stratio/sparkta/plugin/operator/median/MedianOperatorSpec.scala [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.stratio.sparkta.plugin.operator.median
+
+import org.scalatest.{Matchers, WordSpec}
+
+class MedianOperatorSpec extends WordSpec with Matchers {
```
Comment (danielcsant) [inline plugins/operator-median/src/test/scala/com/stratio/sparkta/plugin/operator/median/MedianOperatorSpec.scala]: header anotation @RunWith(classOf[JUnitRunner]) missed

File: plugins/operator-min/src/test/scala/com/stratio/sparkta/plugin/operator/min/MinOperatorSpec.scala [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.stratio.sparkta.plugin.operator.min
+
+import org.scalatest.{Matchers, WordSpec}
+
+class MinOperatorSpec extends WordSpec with Matchers {
```
Comment (danielcsant) [inline plugins/operator-min/src/test/scala/com/stratio/sparkta/plugin/operator/min/MinOperatorSpec.scala]: header anotation @RunWith(classOf[JUnitRunner]) missed

File: plugins/operator-stddev/src/test/scala/com/stratio/sparkta/plugin/operator/stddev/StddevOperatorSpec.scala [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.stratio.sparkta.plugin.operator.stddev
+
+import org.scalatest.{Matchers, WordSpec}
+
+class StddevOperatorSpec extends WordSpec with Matchers {
```
Comment (danielcsant) [inline plugins/operator-stddev/src/test/scala/com/stratio/sparkta/plugin/operator/stddev/StddevOperatorSpec.scala]: header anotation @RunWith(classOf[JUnitRunner]) missed

File: plugins/operator-sum/src/test/scala/com/stratio/sparkta/plugin/operator/sum/SumOperatorSpec.scala [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+ * limitations under the License.
+ */
+
+package com.stratio.sparkta.plugin.operator.sum
+
+import org.scalatest.{Matchers, WordSpec}
+
+import java.math.{BigDecimal => JBigDecimal}
+
+class SumOperatorSpec extends WordSpec with Matchers {
```
Comment (danielcsant) [inline plugins/operator-sum/src/test/scala/com/stratio/sparkta/plugin/operator/sum/SumOperatorSpec.scala]: header anotation @RunWith(classOf[JUnitRunner]) missed

File: plugins/operator-variance/src/test/scala/com/stratio/sparkta/plugin/operator/variance/VarianceOperatorSpec.scala [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.stratio.sparkta.plugin.operator.variance
+
+import org.scalatest.{Matchers, WordSpec}
+
+class VarianceOperatorSpec extends WordSpec with Matchers {
```
Comment (danielcsant) [inline plugins/operator-variance/src/test/scala/com/stratio/sparkta/plugin/operator/variance/VarianceOperatorSpec.scala]: header anotation @RunWith(classOf[JUnitRunner]) missed

File: plugins/operator-avg/src/test/scala/com/stratio/sparkta/plugin/operator/avg/AvgOperatorSpec.scala (hunk 2 above)
Comment (anistal) [inline plugins/operator-avg/src/test/scala/com/stratio/sparkta/plugin/operator/avg/AvgOperatorSpec.scala]: @danielcsant Why? It is extending from WordSpec and @RunWith is not needed.
Furthermore JUnitRunner is a Java's library.

**Pull request merged**